resources.py

```diff
@@ -0,0 +1,7 @@
+#
+#    This is a python library for resources like message templates etc.
+#
```

Comment:
Minor thing: prefer to document Python code in triple-quote blocks. For a module "docstring" I would write it to the top like this:

```
  """
  This module is doing this and that.
  """
```

Similarily for function, class or method, I'd codument it just below the signature, ie:

```
class Foo(object):
  """
  Foo blah..
  """
  def bar(self):
    """
    This is a method docstring
    """
```

Python (and third-party tools) does a great job taking those "docstrings" and generating documentation.